community-fund/COMMUNITY_FUND_POLICY.md

```diff
@@ -41,49 +47,45 @@ Funding is not transferable to another person, or any other event.
 
```

Comment:
Small typo on line 31: https://github.com/openjs-foundation/cross-project-council/pull/1338/files#diff-775d912ef7f0b7103be410d0c4380ff7a5c1b96c057c1f9b28e1044a34cfe14aR31

_All amounts are in United States dollars._

Reply:
Fixed. Thanks!